src/deepqmc/gnn/gnn.py

```diff
@@ -125,13 +125,11 @@ def __init__(
         n_nuc, n_up, n_down = mol.n_particles
         n_atom_types = mol.n_atom_types
         charges = mol.charges
-        coords = mol.coords
         if ghost_coords is not None:
-            coords = jnp.concatenate([coords, jnp.asarray(ghost_coords)])
             charges = jnp.concatenate([charges, jnp.zeros(len(ghost_coords))])
-            n_nuc = len(coords)
+            n_nuc += len(ghost_coords)
             n_atom_types += 1
-        self.coords = coords
+        self.ghost_coords = ghost_coords
```

Comment:
Are ghost co-ordinates not treated as part of the `R` input to the transferable network?

Reply:
Ghost atoms were used in the single-geometry setting to break certain symmetries of the system. In this case it made sense to specify a single set of ghost atoms for the entire training. I see that perhaps in a transferable setting, we'll need to have different ghost atoms for different configurations. Even better would be if we didn't need ghost atoms at all, and managed to find a better way to handle symmetry.

Reply:
Makes sense- I am happy to leave it as it is for now, until we know if we want to have ghost atoms in the transferable version or not